pkg/bootstrap/bootstrap.go

```diff
@@ -27,95 +27,90 @@ import (
 type Phase string
 
 const (
-	Preflight Phase = "preflight"
-	Init      Phase = "init"
-	Addon     Phase = "addon"
+	Preflight  Phase = "preflight"
```

Comment:
Preflight: preflight checks;
Init: initilaization;
Postflight: what does postflight for?

Reply:
Postflight is mainly reserved, and some operations are performed after initialization.